### ISSUE TITLE
What colorscheme does it use or how to change it

### ISSUE THREAD

**Issue by milasudril** [Swordfish90/cool-retro-term#536]
I would like to make it look like CGA. This should be Linux-colors I think. By default, it looks ubuntu-like. I see no option to change the color scheme.

**Comment by GabrielRRussell** [comment]
If you click on the menubar option for Edit, then go to Settings, you can find all of the preference options.
Select a profile you'd like to modify, or create a new one. Then go to the other tabs and mess with everything you see fit.

If you'd like to use a prebuilt theme, just click Profiles and select one. 

If you've disabled the Menu bar, you can hold right click on the console to bring it up.

**Comment by milasudril** [comment]
I am looking for something like this:

![konsole](https://user-images.githubusercontent.com/734628/60116880-3fe01d80-9779-11e9-9302-d8fd36f1a8d1.png)

**Comment by Swordfish90** [comment]
Hi @milasudril ... Sadly this is currently not possible. You can customize foreground and background color but the colorScheme is set in:

https://github.com/Swordfish90/cool-retro-term/blob/master/app/qml/PreprocessedTerminal.qml

You can change it there, but there are no plan to support it via GUI at the moment.

**Comment by milasudril** [comment]
Now that I have found the loaded file I can change it manually. Maybe there could be some option for setting the file. Anyway:

    color 2 0 0 0 0 0 # regular color 0 Black
    color 3 170 0 0 0 0 # regular color 1 Red
    color 4 0 170 0 0 0 # regular color 2 Green
    color 5 170 85 0 0 0 # regular color 3 Yellow "Brown"
    color 6 0 0 170 0 0 # regular color 4 Blue
    color 7 170 0 0 0 0 # regular color 5 Magenta
    color 8 0 170 170 0 0 # regular color 6 Cyan
    color 9 170 170 170 0 0 # regular color 7 White

    color 12 85 85 85 0 0 # intensive color 0
    color 13 255 85 85 0 0 # intensive color 1
    color 14 85 255 85 0 0 # intensive color 2
    color 15 255 255 85 0 0 # intensive color 3
    color 16 85 85 255 0 0 # intensive color 4
    color 17 255 85 255 0 0 # intensive color 5
    color 18 85 255 255 0 0 # intensive color 6
    color 19 255 255 255 0 0 # intensive color 7

This gives the look of an mid 80's early 90's DOS PC in VGA mode. Do you use a 9x14 font or the later 9x16 font? The zero indicates the former.

![vga](https://user-images.githubusercontent.com/734628/60197792-6f0d9200-9840-11e9-98f5-f958f0d4b393.png)

**Comment by Swordfish90** [comment]
Nice! You are using the scanlined version which should be the 1981 8x8 variant of that font.
You can also find the 8x16 variant, but you need to set "Rasterization" to "Default" in order to use it.

Here's the specific link to those fonts if you want to play with them:

https://int10h.org/oldschool-pc-fonts/fontlist/

**Comment by altruios** [comment]
this looks fantastic: and would be a great addition!

**Comment by khizka** [comment]
this is amazing, thank you

**Comment by exoess** [comment]
> Hi @milasudril ... Sadly this is currently not possible. You can customize foreground and background color but the colorScheme is set in:
> 
> https://github.com/Swordfish90/cool-retro-term/blob/master/app/qml/PreprocessedTerminal.qml
> 
> You can change it there, but there are no plan to support it via GUI at the moment.

how do you actually change the colors tho?

**Comment by rmvpress** [comment]
> > Hi @milasudril ... Sadly this is currently not possible. You can customize foreground and background color but the colorScheme is set in:
> 
> > 
> 
> > https://github.com/Swordfish90/cool-retro-term/blob/master/app/qml/PreprocessedTerminal.qml
> 
> > 
> 
> > You can change it there, but there are no plan to support it via GUI at the moment.
> 
> 
> 
> how do you actually change the colors tho?

Yeah would anyone be able to do a guide or something, I have this program running on Ubuntu and I've looked through all the directories that have cool-retro-term in the name and their given subdirectories and cannot locate this file so I can edit it

**Comment by kyagloski** [comment]
> Now that I have found the loaded file I can change it manually. Maybe there could be some option for setting the file. Anyway:
> 
> ```
> color 2 0 0 0 0 0 # regular color 0 Black
> color 3 170 0 0 0 0 # regular color 1 Red
> color 4 0 170 0 0 0 # regular color 2 Green
> color 5 170 85 0 0 0 # regular color 3 Yellow "Brown"
> color 6 0 0 170 0 0 # regular color 4 Blue
> color 7 170 0 0 0 0 # regular color 5 Magenta
> color 8 0 170 170 0 0 # regular color 6 Cyan
> color 9 170 170 170 0 0 # regular color 7 White
> 
> color 12 85 85 85 0 0 # intensive color 0
> color 13 255 85 85 0 0 # intensive color 1
> color 14 85 255 85 0 0 # intensive color 2
> color 15 255 255 85 0 0 # intensive color 3
> color 16 85 85 255 0 0 # intensive color 4
> color 17 255 85 255 0 0 # intensive color 5
> color 18 85 255 255 0 0 # intensive color 6
> color 19 255 255 255 0 0 # intensive color 7
> ```
> 
> This gives the look of an mid 80's early 90's DOS PC in VGA mode. Do you use a 9x14 font or the later 9x16 font? The zero indicates the former.
> 
> ![vga](https://user-images.githubusercontent.com/734628/60197792-6f0d9200-9840-11e9-98f5-f958f0d4b393.png)

how do you set these variables in PreprocessedTerminal.qml? Sorry, I have never even heard of qml before this point.
